Update to Create 6.0.7+ and VS 2.4+

chj990707 opened the pull request:
This PR is heavily under construction so expect bugs.

chj990707 committed:
Updating to VS2.3 beta 11 and Create 6.0.7+.
FlwContraption is removed, so the mixin needs to be moved somewhere else (probably ContraptionVisual or ClientContraption). Actors aren't currently rendered correctly.
Cannot right-click interact or place the blocks on a interact-me contraption, but things like taking dropped items in, kinetics, item processing seems to work.

chj990707 committed:
Oh god. This commit is a mess.
- Updated the dependency version of create and removed the transitive one from VS2.
- Fixed interaction/placing blocks on interact-me contraptions.
- Fixed rendering of interact-me contraptions, both with and without flywheel backend.

chj990707 committed:
Mounted Storage system is fixed in interactive contraptions.

chj990707 committed:
Upporting to 2.4, things that need to be checked:
- ClientShip render transform update. Where did the method go?
- Change of carriage constraints to joints.
- Disabling Contraption Ship Kinematic velocity and portal dimension change.

chj990707 committed:
Propagator visual is registered via Visualizer Registry

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/ValkyrienSkies/Create-Interactive?pullRequest=32) <br/>All committers have signed the CLA.

chj990707 commented:
Known issues:
- Shafts on propagator bearing does not render.
- Placing disjointed bearing crashes the game client-side.

blockninja124 committed:
Fix mixin remaps

chj990707 committed:
Disjointed Propagator crashing the game was fixed.
- using isWindmill as property crashes the game. Why didn't it prior to update?

StewStrong commented:
Pretty based if I say so myself 😃

chj990707 committed:
Fixed the logging to be more descriptive.

chj990707 committed:
Fixed interaction with train controls.
- the interaction should be cancelled/fail when it did something with contraption.

chj990707 committed:
derailed train joints bug resolved. forgot to set the joint value to null when it was deleted.

chj990707 committed:
Fixed local transformation angles transformed to degrees when it already is

chj990707 committed:
Instead of applying superglue upon placement, interact-me sticker is now face attached block and acts like a sign, attaching itself to the contraption automatically.

chj990707 committed:
Deployer on interactive contraptions/trains rendering dupe fixed.

blockninja124 committed:
Update VS

blockninja124 committed:
IhateremapIhateremapIhateremap

chj990707 committed:
Fixed players having their rotation doubled when seated on an interactive contraption.

chj990707 committed:
bump to 2.4.7

chj990707 committed:
Disabled neighbor blocks updating upon interactive contraption blocks copied to the shadow ship.

chj990707 committed:
Bump to VS 2.4.8
Changed dependency range to VS

chj990707 committed:
Fixed Fabric server crashing from client-only method mixin.

chj990707 committed:
Added max joint distance to train derailing

blockninja124 commented:
I already released this to MR/CF but forgot to merge the PR :facepalm:
